alpha/lib/model/index/kuserIndex.php

```diff
@@ -73,6 +73,8 @@ public static function getIndexFieldsMap()
 				'permission_names' => 'indexedPermissionNames',
 				'role_ids' => 'indexedRoleIds',
 				'type' => 'type',
+				'company' => 'company',
```

Comment:
Is this data needed to be indexed to sphinx as well? 
If not why was it added here?